src/development/LocalTest/Services/LocalApp/Implementation/LocalAppHttp.cs

```diff
@@ -44,7 +53,7 @@ public LocalAppHttp(IHttpClientFactory httpClientFactory, IOptions<LocalPlatform
                 cacheEntry.SetSlidingExpiration(TimeSpan.FromSeconds(5));
                 return await _httpClient.GetStringAsync($"{appId}/api/v1/applicationmetadata?checkOrgApp=false");
             });
-            return JsonConvert.DeserializeObject<Application>(content);
+            return JsonSerializer.Deserialize<Application>(content, new JsonSerializerOptions{PropertyNameCaseInsensitive = true} );
```

Comment:
Maybe consider caching the deserialized Application object instead of the file content..

Reply:
I did consider that, but I tend to favor not caching mutable objects. It would be pretty silly for someone to mutate the Application object, so this is mostly just following my normal pattern.